man/systemd.resource-control.xml

```diff
@@ -691,6 +691,26 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AllowBindPorts=<replaceable>IP_PORT</replaceable></varname></term>
+
+        <listitem>
+          <para>Allow binding a socket address only to certain IP ports specified by <replaceable>IP_PORT</replaceable>.
+          Same ports will be allowed for both IPv4 and IPv6 protocols. Port value must be in [0, 65535] range.
+          Special value <constant>none</constant> is reserved to block binding to any port.
+          </para>
+
+          <para><varname>AllowBindPorts</varname> is powered by BPF program compiled from source code and supported only
```

Comment:
we typically suffix our setting names with `=` in the docs, to make clear they are settings. hence please write `AllowBindPorts=` instead of just `AllowBindPorts` here